crates/nu-parser/src/parser.rs

```diff
@@ -663,8 +665,12 @@ pub fn parse_multispan_value(
         SyntaxShape::Expression => {
             trace!("parsing: expression");
 
-            let (arg, err) =
-                parse_expression(working_set, &spans[*spans_idx..], expand_aliases_denylist);
+            let (arg, err) = parse_expression(
+                working_set,
+                &spans[*spans_idx..],
+                expand_aliases_denylist,
+                false, // not sure, make it to false to keep behavior unchanged
```

Comment:
What is the issue here?

Reply:
It's in `parse_multispan_value` function, it mainly parsed a multi-arguments as a single argument, an example: `﻿﻿ modified >= (date now) - 2wk`.

For the expression like this, I'm not sure if the expression itself is a subexpression, so I decide to make it false, to avoid change behavior